xdsl/transforms/lower_csl_wrapper.py

```diff
@@ -311,9 +320,11 @@ def match_and_rewrite(self, op: csl_wrapper.ImportOp, rewriter: PatternRewriter,
         import_ = csl.ImportModuleConstOp(
             op.module, structs[-1] if len(structs) > 0 else None
         )
+        import_.result.name_hint = self._get_name_hint(op)
 
         rewriter.insert_op(ops, InsertPoint.at_start(csl_mod.body.block))
-        rewriter.replace_matched_op([*structs, import_])
+        rewriter.insert_op(structs, InsertPoint.before(op))
+        rewriter.replace_matched_op(import_)
```

Comment:
This is to prevent the structs from having the same name as the buffer, and the buffer turning into `buf_8` for instance, which may be confusing.